### PR TITLE
build a custom aarch C compiler

### PR DESCRIPTION
**Pull request by japaric**
to lower our glibc min version requirement

r? @alexcrichton

### CODE REVIEW

File: slaves/linux-cross/Dockerfile [hunk 1]
```diff
@@ -101,10 +103,17 @@ USER root
...
 # don't have this name in the component by default either.
+# Also the aarch64 compiler is prefixed with `aarch64-unknown-linux-gnueabi`
+# by crosstool-ng, but Ubuntu just prefixes it with `aarch64-linux-gnu` so
+# we'll, additionally, strip the eabi part from its binaries.
 RUN                                                                           \
   for f in `ls /x-tools/*-unknown-linux-*/bin/*-unknown-linux-*`; do          \
     g=`basename $f`;                                                          \
     ln -vs $f /usr/bin/`echo $g | sed -e 's/-unknown//'`;                     \
+  done && \
+  for f in `ls /usr/bin/aarch64-linux-gnueabi-*`; do                          \
```
Comment (alexcrichton) [inline slaves/linux-cross/Dockerfile]: Should this be from /x-tools instead of /usr/bin?

Comment (japaric) [inline slaves/linux-cross/Dockerfile]: The first `for` created these symlinks:

```
/usr/bin/aarch64-linux-gnueabi-gcc -> /x-tools/aarch64-unknown-linux-gnueabi/bin/aarch64-unknown-linux-gnueabi-gcc
```

The second `for` renames these links to remove the `eabi` part.

Comment (alexcrichton) [inline slaves/linux-cross/Dockerfile]: lol right

**Comment by alexcrichton** [comment]
Ok, I'm now pushing up `alexcrichton/rust-slave-linux-cross:2016-03-29` with this tag.

**Comment by alexcrichton** [comment]
er, with this change*

**Pull request merged**